### ISSUE TITLE
[WhenToStopFuzzing] Enigma Machine picture width attribute breaks Latex export to PDF

### ISSUE THREAD

**Issue by alanvivona** [uds-se/fuzzingbook#45]
**Describe the bug**
The export to PDF fails for the following notebook: WhenToStopFuzzing

**To Reproduce**
1. Go to WhenToStopFuzzing notebook
2. Export to PDF
3. Cry because you can't export the notebook

**Expected behavior**
Export should render and download a PDF file

**Screenshots**
Won't add. I'm sending a fix in a second

**Desktop (please complete the following information):**
Irrelevant, as far as I know

**Additional context**
No need

**Comment by alanvivona** [comment]
Pushed a quick fix here: https://github.com/uds-se/fuzzingbook/pull/46

**Comment by alanvivona** [comment]
Just to be 100% clear: I read the FAQ, yes... but the default PDF export from the Jupyter notebook's GUI is already working. I  printed out some chapters yesterday and while I was doing so I found out about this issue, went ahead and fixed it in order to be able to print the "WhenToStopFuzzing" notebook too.
I know you are not actively supporting PDF export but since it's already working (except for this particular notebook) I thought it might be useful for you guys to get this fixed so others can print this notebook as I did. That's why I opened the PR with the fix.

Awesome resource btw, I'm having fun reading it. Appreciate the effort of writing all this and making it open source. Thanks!
